user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_triggering_DOM_midnight_sync.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: SLNetClientTest_triggering_DOM_midnight_sync
```

Comment:
When being on this page, no item is selected on the treeview on the left. I guess the toc.yml needs to be updated?